CHANGELOG.md

```diff
@@ -4,6 +4,26 @@
 
 ## DFX
 
+### feat: deploy to playground
+
+Introduced a new network type called `playground`. Canisters on such networks are not created through standard means, but are instead borrowed from a canister pool.
+The canisters time out after a while and new canisters need to be borrowed for further deployments.
+To define custom playground networks, use a network definition that includes the `playground` key:
+```json
+"<network name>": {
+  "playground": {
+    "playground_canister": "<canister pool id>",
+    "timeout_seconds": <amount of seconds after which a canister is returned to the pool>
```

Comment:
it is not clear if there is an upper limit to `timeout_seconds`

Reply:
I guess `u64::MAX` is a limit, but it's more on the pool to define the correct value